.github/workflows/test.yml

```diff
@@ -1,12 +1,25 @@
 name: Test
 
-on: [push, pull_request]
+on:
+  - push
+  - pull_request
 
 jobs:
   test:
     strategy:
       matrix:
-        toolchain: ["1.60.0", "stable", "beta", "nightly"]
+        toolchain:
+          - "1.60.0"
+          - "stable"
+          - "beta"
+          - "nightly"
+        features:
```

Comment:
> I'd be ok with suppressing warnings from (or skipping) `cargo doc` on `no_std`.

Sure! I opted to enable `std` when building the docs. I also addressed the compiler warnings.

After mucking around workflow matrices, please let me know your thoughts on the number of parallel jobs that are being run. Given that feature matrices are inherently multiplicative, would you prefer that some features are omitted so that CI doesn't have to run too many jobs?